src/main.cpp

```diff
@@ -1,6 +1,87 @@
-#include <bits/stdc++.h>
-
+#include <string>
+#include <vector>
+#include <sstream> //istringstream
+#include <iostream> // cout
+#include <fstream> // ifstream
+ 
 using namespace std;
+ 
+vector<vector<int>> tilesScenario;
+
+/**
+ * Reads csv file into table, exported as a vector of vector of ints.
+ * @param inputFileName input file name (full path).
+ * @return data as vector of vector of ints.
+ */
+vector<vector<int>> parse2DCsvFile(string inputFileName);
+int getTile(unsigned int line, unsigned int column, unsigned int depth);
+int getTileXYZ(unsigned int x, unsigned int y, unsigned int z);
+
 int main()
 {
+    tilesScenario = parse2DCsvFile("config/avalon-main-map.csv");
+ 
+    // for (auto l : tilesScenario) {
+    //     for (auto x : l){
+    //         cout << x << " ";
+    //     }       
+    //     cout << endl;
+    // }
+ 
+    cout << getTile(2,3,1) << endl;
+    cout << getTile(2,4,2) << endl;
+
+    cout << getTileXYZ(0,0,0) << endl;
+    cout << getTileXYZ(1,1,0) << endl;
+    cout << getTileXYZ(2,2,0) << endl;
+    
+    return 0;
+}
+
+vector<vector<int>> parse2DCsvFile(string inputFileName) {
+    vector<vector<int> > data;
+    ifstream inputFile(inputFileName);
+    int l = 0;
+    while (inputFile) {
+        l++;
+        string s;
+        if (!getline(inputFile, s)) break;
+        if (s[0] != '#') {
```

Comment:
```suggestion
        if (s[0] == '#')
            continue;
```
se for verdadeiro tu pula pra próxima iteração. Não precisa indentar o bloco todo dai